src/codegen/templates/library_interpreter_templates/default_c_function_call.py.mako

```diff
@@ -0,0 +1,25 @@
+<%page args="function"/>\
+<%
+    from codegen.utilities.interpreter_helpers import generate_interpreter_function_call_args
+    from codegen.utilities.function_helpers import get_arguments_type
+    from codegen.utilities.text_wrappers import wrap, docstring_wrap
+%>\
+
+        cfunc = lib_importer.${'windll' if function.calling_convention == 'StdCall' else 'cdll'}.DAQmx${function.c_function_name}
```

Comment:
Please remove the extra blank line above `cfunc = ...` where the docstring would go